src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+@import "../normalize/normalize.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: 7 * $cell-size + 10 * $cell-gap;
```

Comment:
it's a perfect place for function 
and I would suggest using max-width instead width